src/backend/joanie/demo/management/commands/create_dev_demo.py

```diff
@@ -114,7 +117,9 @@ def create_product_credential(self, user, organization, batch_size=1):
             for i in range(batch_size)
         ]
 
-    def create_product_certificate(self, user, organization, batch_size=1):
+    def create_product_certificate(
+        self, user, organization, contract_definition=None, batch_size=1
```

Comment:
I don't think it's a good idea to link contract definition to certificate product for now.